### DISCUSSION TITLE
Can it be used in InvokeAi?

### DISCUSSION THREAD

**Discussion by halilergul1** [hako-mikan/sd-webui-regional-prompter#236] [Announcements]
Hi All,

Did anyone try to integrate-implement this nice extension in invoke-ai engine? That would be nice to see as I currently working on it

**Comment by Symbiomatrix** [comment]
Hello. Sorry, I'll pass. Fiddling with webui's madness has been more than enough for me.

Edit: Case in point, torch 2.0.0...

**Comment by JaysonPhoenix** [comment]
i was gonna ask for the same thing, here and at invokes github. because composable prompt kind of extensions or (however its called) like yours and the cutoff extension for example are becoming as mandatory as controlnet, its just that less people know about em yet and its more complicated to use em, but once people use em they cant go without em since it adds great possibilities and provides workarounds for some of stable diffusions greatest flaws in general. and i would want invoke to hard implement at least all mandatory tools into their ui since they dont want to go for extension support in general. also invoke has kind of its own language which i see as a potential to implement non gui related stuff like cutoff or part of regional prompter (prompt mask for example) right into the language while their advanced/modern ui allows for proper visualization of gui related stuff like creating 2d regions and defining masks. like imagine how cool it would be to have a stable diffusion language understand some kind of marks or whatever to properly seperate token influences to not have color contamination right out the box or to properly seperate lora influences for certain tokens or parts of a picture, would be pretty cool. i see invoke as kind of advanced tool for stable diffusion doing stuff proper and stable with a user friendly modern ui but later, while automatics advantage is it always sitting on the bleeding edge of workflow/possibility advancement through its extensions but beeing unstable as hell and has a clunky patchwork ui.

so id ask would you please think about it again and maybe work together with invoke guys to implement your great extension into invoke too?

also what did you mean with that torch 2.0.0 comment? regional prompter havin trouble with torch 2? because i didnt update it yet beeing afraid of stuff not mentioned not working anymore.
